printer/dot/printer.go

```diff
@@ -33,14 +42,55 @@ func (d Dot) Print(g *graph.Graph, w io.Writer) error {
 		if pv == nil {
 			pv = provider.RawProvider{}
 		}
-		shape := "ellipse"
+		attr := map[string]string{
```

Comment:
`attr` could be defined outside the loop, with the common params:
```go
			attr["shape"] = "plaintext"
			attr["labelloc"] = "b"
			attr["height"] = "1.15"
```
Only `shape` and `image` are variables

Reply:
Nop, those are only set if `the Resource has image` and `Options.ShowIcons`, and where is it now defined is the only place where this things are fulfilled. If not those should not be seted.